redux/ducks/articles.js

```diff
@@ -36,13 +38,21 @@ const currentReducer = defaultReducer((state, { payload }) => ({
   pending: false,
 }));
 
+const FIRST_PAGE_NUMBER = 0;
+
 export default createReducer(
   {
     [FETCH_ALL]: defaultReducer((state, { payload: { data } }) => ({
       ...state,
       data,
       pending: false,
     })),
+    [FETCH_CHUNK]: defaultReducer((state, { payload: { data, next } }) => ({
+      ...state,
+      data: next.page === FIRST_PAGE_NUMBER + 1 ? data : [...state.data, ...data],
```

Comment:
what about init data with `[]` & omit this ternary?

Reply:
@Drapegnik , this action is dispatched when you are navigating across different tabs or come to the main page through Back button. In a such case the store isn't reloaded.
I could try to reload it as you do here https://github.com/babajka/babajka-frontend/blob/master/routes.js#L36 or leave code as it is